src/bacnet/basic/object/msv.c

```diff
@@ -328,7 +372,43 @@ bool Multistate_Value_State_Text_Set(
     }
 
     return status;
-    ;
+}
+
+/**
+ * Initialize the Multistate Value State Text Inputs. Returns false if there are errors.
+ *
+ * @param pInit_state_text_data pointer to  state text initialisation values
+ *
+ * @return true/false
+ */
+bool Multistate_Value_Set_State_text_init(MSV_STATE_TEXT_INIT_OPTIONS_LIST *pInit_state_text_data) {
+
+    unsigned int option_index = 0;
+    unsigned int i = 0;
+    unsigned int j;
+
+    if((int) pInit_state_text_data->length > MSV_Max_Index) {
+        PRINT("pInit_state_text_data->length = %d >= %d", (int) pInit_state_text_data->length, MSV_Max_Index);
+        return false;
+    }
+
+    for (i = 0; i <= MSV_Max_Index; i++) {
+
+        if(i == MSV_Max_Index) {
```

Comment:
sometimes you insert a ' ' after if or for , sometimes you do not. Please try being consistent.

Reply:
added in the space